### PR TITLE
Full availability api

### LINKED ISSUE #292: Refactor record view
* Move the thumbnail to the left side of the screen
 * Move the metadata slightly right to make room for the above
 * Move the availability window up to the top

I.E.:
 
md_layout_refactor

### LINKED ISSUE #302: Availability labels
Given: http://getbootstrap.com/components/#available-variations

I propose we use light blue (`label-info`) for electronic resources and orange (`label-warning`) for "Discharged" and "Non-Circulating". 

Wait until pulibrary/marc_liberation#39 and pulibrary/marc_liberation#40 are done to avoid later refactoring.

### LINKED ISSUE #319: Availability Labels and Logic
https://docs.google.com/spreadsheets/d/1MVM4_Q2ZzFnrpzWtEGywnAoB5hndySOgFzr4Hfun200/edit#gid=979200987

### PR DESCRIPTION
**Pull request by tampakis**
- Matches Voyager status to simplified labels. Closes #319 and closes #302 (Note that for all rare books locations to appear as "on-site access" I need to make a change to the bibdata availability API.
 - Have title row and bibliographic info span 9 columns and availability panel span 3 columns. This allows for a better alignment of RTL fields along the left edge of the availability panel. Finally closes #292.

### CODE REVIEW

File: spec/features/availability_spec.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@
       visit "/catalog/8606339"
       sleep 5.seconds
       expect(page.all('.location--online .umlaut .fulltext').length).to eq 1
-      expect(page.all('.location--online .umlaut .fulltext .response_item').length).to eq 5
+      expect(page.all('.location--online .umlaut .fulltext .response_item').length).to be >= 4
```
Comment (kevinreiss) [inline spec/features/availability_spec.rb]: This test was poor form on my part. Should not have tied it to a number.

**Comment by kevinreiss** [comment]
:+1: Looks good.

File: spec/features/availability_spec.rb (hunk 1 above)
Comment (tampakis) [inline spec/features/availability_spec.rb]: The weird thing was getit-dev retrieved only 4 links but getit retrieves 5.

Comment (kevinreiss) [inline spec/features/availability_spec.rb]: That is my fault. I have a slightly different configuration for the app on getit-dev and getit for a feature collapses duplicate listings from the same vendor. Getit-dev is more generous in this match. Sorry for the confusion.

**Pull request merged**